packages/core/src/utils/format.ts

```diff
@@ -8,8 +8,8 @@ export const formatEta = (ms: number) => {
   const s = seconds - h * 3600 - m * 60;
 
   const hstr = h > 0 ? `${h}h ` : "";
-  const mstr = m > 0 || h > 0 ? `${m}m ` : "";
-  const sstr = s > 0 || m > 0 ? `${s}s` : "";
+  const mstr = m > 0 || h > 0 ? `${m < 10 ? "0" : ""}${m}m ` : "";
```

Comment:
might only be worth prefixing if `h > 0`

```suggestion
  const mstr = m > 0 || h > 0 ? `${m < 10 && h > 0 ? "0" : ""}${m}m ` : "";
```

Reply:
Yea I think it looks better without the leading zero.

before
```
│     ETA │
┼─────────┤
│ 03m 08s │
```

after
```
│     ETA │
┼─────────┤
│  3m 08s │
```